biomage-org/ui
# [BIOMAGE-2402] - Allow upload of prefixed samples

# Description
Allow upload of prefixed samples.

To test: 
- Upload normal files -> should pass
- Upload prefixed files (e.g. KO_barcodes.tsv.gz, etc.) -> should pass
- Upload postfixed files (e.g. barcodes_KO.tsv.gz, etc.) -> should fail

# Details
#### URL to issue
https://biomage.atlassian.net/browse/BIOMAGE-2402

#### Link to staging deployment URL (or set N/A)
https://ui-agi-ui187.scp-staging.biomage.net/

#### Links to any PRs or resources related to this PR
N/A

#### Integration test branch
master

# Merge checklist
Your changes will be ready for merging after all of the steps below have been completed.

### Code updates
Have best practices and ongoing refactors being observed in this PR

### Manual/unit testing

### Integration testing
**You must check the box below** to run integration tests on the latest commit on your PR branch.
Integration tests have to pass before the PR can be merged. Without checking the box, your PR
**will not pass** the required status checks for merging.


### Documentation updates

### Optional
